src/Interface/Modules/Render/ViewScene.cc

```diff
@@ -47,7 +47,7 @@ using namespace SCIRun::Render;
 ViewSceneDialog::ViewSceneDialog(const std::string& name, ModuleStateHandle state,
 	QWidget* parent /* = 0 */)
 	: ModuleDialogGeneric(state, parent), mConfigurationDock(0), shown_(false),
-	itemManager_(new ViewSceneItemManager)
+	itemManager_(new ViewSceneItemManager(state, this))
```

Comment:
Passing this in initialization list of constructor is risky. I'd pull that out and use a setter inside the constructor body